### ISSUE TITLE
Indicate maximum attunement can be edited in edit mode

### ISSUE THREAD

**Issue by etiquettestartshere** [foundryvtt/dnd5e#2991]
![image](https://github.com/foundryvtt/dnd5e/assets/148253744/6dbc72da-a880-446d-b066-b44ef9065789)
![image](https://github.com/foundryvtt/dnd5e/assets/148253744/6f8df5f4-c4a8-4765-8983-26d0ccf6cbd4)
When in edit mode, it is unclear that this value can be edited, and the greyed out text implies (at least to me) that it cannot be (it only becomes evident on hover).

If it changed to black in edit mode it would be more clear--another option would be to make the input box visible, like name or xp.
